docs/clients/dotnet/21.2/connecting/compatibility-mode.md

```diff
@@ -0,0 +1,66 @@
+# Compatibility Mode
+
+Compatibility Mode was added in v21.2 of the client to set certain configuration options depending on the Event Store version the client is connecting to.
+
+At the moment, the v21.2 client supports two modes:
+
+- "5" to allow connection to Event Store servers v5.0.10.
```

Comment:
```suggestion
- "5" to allow connection to v5 Event Store servers.
```
Auto-compatibility is compatible with any version of v5

Reply:
Fixed in https://github.com/EventStore/documentation/pull/288/commits/151543c1acbb559165d3d983279156be408f5eb2.